recipes/cpr/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "cpr" %}
+{% set version = "1.12.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/libcpr/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: f64b501de66e163d6a278fbb6a95f395ee873b7a66c905dd785eae107266a709
+  patches:
+    # conda-forge's curl uses autotools, so we cannot use cmake COMPONENTS
+    - patches/find_package_curl.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]
```

Comment:
Is there any reason for not building the library on other operating systems? IF there is an upstream limitation, please link it here in a comment, thanks!

Reply:
It is not requirement, but it is common courtesy to explain the motivations behind the PR as part of the PR, so what you wrote is perfectly fine.